ILSwiss/rlkit/torch/algorithms/sac/sac_alpha.py

```diff
@@ -79,7 +81,10 @@ def train_step(self, batch):
         # policy_params = itertools.chain(self.policy.parameters())
 
         rewards = self.reward_scale * batch["rewards"]
-        terminals = batch["terminals"]
+        if self.ignore_terminal:
+            terminals = 0.0
+        else:
+            terminals = batch["terminals"]
```

Comment:
Same as in `sac.py`

Reply:
I tried to use this option to alleviate the terminal effect during training. Not I have removed the option.